tests/integration-test.sh

```diff
@@ -22,24 +22,25 @@ dpkg -l > package.list
 
 
 buildinfo=$(drone build info vmware/vic $DRONE_BUILD_NUMBER)
+export GOVC_TLS_HANDSHAKE_TIMEOUT=30s
 
 if [[ $DRONE_BRANCH == "master" || $DRONE_BRANCH == "releases/"* ]] && [[ $DRONE_REPO == "vmware/vic" ]] && [[ $DRONE_BUILD_EVENT == "push" ]]; then
     echo "Running full CI for $DRONE_BUILD_EVENT on $DRONE_BRANCH"
-    pybot --removekeywords TAG:secret --exclude skip tests/test-cases
+    pabot --processes 2 --removekeywords TAG:secret --exclude skip tests/test-cases
```

Comment:
parameterize the number of processes or do we want to be able to chose per-scenario values?